Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
Is it necessary to offer such flexibility? Or does pandoc version matters a lot?
I don't think we should make the Makefile complex to meet such needs, I think people who really want to use their own local pandoc can use it directly without Makefile.